filebeat/docs/autodiscover-hints.asciidoc

```diff
@@ -100,6 +100,33 @@ Define an ingest pipeline ID to be added to the {beatname_uc} input/module confi
 co.elastic.logs/pipeline: custom-pipeline
 -----
 
+When hints are used along with templates, then hints will be evaluated only in case
+there is no template's condition that resolve to true. For example:
```

Comment:
```suggestion
there is no template's condition that resolves to true. For example:
```

should here maybe be added clarification if this is evaluated per container basis?

Reply:
At this docs section we refer in general in docker+kubernetes autodiscover hence I would avoid mixing the concept of event based evaluation per container/pod for simplicity. In addition the way that autodiscover works is more or less described on the first paragraph of this page.